kevm

```diff
@@ -42,6 +42,7 @@ run_env() {
     export PATH="$release_dir/lib/native/linux:$release_dir/lib/native/linux64:$release_dir/bin/:$PATH"
     export LD_LIBRARY_PATH="$release_dir/lib/native/linux64:$lib_dir:${LD_LIBRARY_PATH:-}"
     eval $(opam config env)
+    export PATH="$build_dir/kore/.stack-work/install/x86_64-linux/lts-11.3/8.2.2/bin/:$PATH"
```

Comment:
This should not be necessary. Stack should set the correct `PATH` on its own. It is also very fragile; we have already updated Kore to use a newer Stackage resolver and GHC version, so this will break the next time you update.

Reply:
Good thought, thanks. I pushed a commit that improves this.